### DISCUSSION TITLE
Cannot launch 0.28.1?

### DISCUSSION THREAD

**Discussion by lumynou5** [kovidgoyal/kitty#6453] [Q&A]
I installed kitty 0.28.1 on Arch Linux and cannot launch it at all. The shell also complains that it's unable to find a terminal to launch. However 0.27.1 works well.
How to solve this?

**Comment by kovidgoyal** [comment]
Run kitty inside some other terminal. Once you see it works, install a .desktop file or whatever mechanism your desktop environmnet needs to inform it of your preferred terminal emulator
